components/eamxx/src/physics/shoc/eamxx_shoc_process_interface.cpp

```diff
@@ -230,6 +230,11 @@ void SHOCMacrophysics::init_buffers(const ATMBufferManager &buffer_manager)
 // =========================================================================================
 void SHOCMacrophysics::initialize_impl (const RunType run_type)
 {
+  // Gather runtime options
+  runtime_options.lambda_low    = m_params.get<Real>("lambda_low",0.001);
```

Comment:
I would not provide a default here. Instead, I would put a default value in the XML file. Users won't know about this default just looking at the docs, but they will see any default value that is present in the XML file.